Documentation/ceph-cluster-crd.md

```diff
@@ -35,14 +35,18 @@ spec:
   dataDirHostPath: /var/lib/rook
   mon:
     count: 3
-    allowMultiplePerNode: true
+    allowMultiplePerNode: false
```

Comment:
What about just removing the flag if `false`?

Reply:
My thought is that it should be very clear in the docs that by default we don't allow multiple mons per node.

Reply:
Ok